mkdocs/docs/tutorials/load-test-http/appserverlessml.md

```diff
@@ -0,0 +1,60 @@
+---
+template: main.html
+---
+
+# K8s App/serverless/ML Frameworks
+
+This tutorial provides examples of using the `load-test-http` experiment chart with various Kubernetes app/serverless/ML frameworks. Refer to [`load-test-http` usage](usage.md) to learn more about this chart.
+
+!!! tip "Dear Iter8 community" 
+
+    These examples are maintained by members of the Iter8 community, and may become outdated. If you find that something is not working, lend a helping hand and fix it in a PR. More examples are always welcome.
+
+## Knative
+
+???+ note "Before you begin"
+    1. [Install Iter8](../../getting-started/install.md).
+    2. [Install Knative and deploy your first Knative Service](https://knative.dev/docs/getting-started/first-service/). As noted at the end of the Knative tutorial, when you curl the Knative service,
+    ```shell
+    curl http://hello.default.127.0.0.1.sslip.io
+    ```
+    you should see the expected output as follows.
+    ```
+    Hello World!
+    ```
+    3. Download experiment chart.
+    ```shell
+    iter8 hub -e load-test-http
+    cd load-test-http
+    ```
+
+### 1. Run experiment
```

Comment:
same comment about terseness as in http tests

Reply:
Could we please revisit this in v0.9? I would like your first Knative experiment to run inside K8s actually (so that it can support both external and cluster local ksvcs).

Reply:
Done.